modules/developers-guide/pages/authnz.adoc

```diff
@@ -97,8 +97,7 @@ Once these steps are completed, queries to Stargate can use the generated token
 `X-Cassandra-Token` for either cURL commands using the REST or Document API, or
 for GraphQL mutations.
 
-TIP: https:///astra.datastax.com[Astra, DataStax's Database-as-a-Service], is backed
-by Keycloak internally, and roles and permissions can be implemented with Astra's dashboard.
+TIP: You can define roles and permissions in the https:///astra.datastax.com[Astra DB console]. Astra DB is DataStax's cloud-native database.
```

Comment:
I think this edit loses some important context for the tip. How about this?
```suggestion
TIP: If you are using Stargate as part of DataStax's Astra DB cloud-native database, roles and permissions are defined in the https://astra.datastax.com/[Astra console].
```

Also, you have three slashes in your link URL.